sources/native/Stride.Native.targets

```diff
@@ -39,9 +45,9 @@
   
   <!-- Define default CPU architectures -->
   <ItemGroup Condition="'$(TargetFramework)' == '$(StrideFramework)'">
-    <StrideNativeCPU Include="win-x64;win-x86" LibraryExtension=".dll" LibraryRuntime="win" />
+    <StrideNativeCPU Condition="$([MSBuild]::IsOSPlatform('Windows'))" Include="win-x64;win-x86" LibraryExtension=".dll" LibraryRuntime="win" />
     <StrideNativeCPU Include="linux-x64"  LibraryExtension=".so" LibraryRuntime="linux" />
-    <StrideNativeCPU Include="osx-x64"  LibraryExtension=".dylib" LibraryRuntime="osx" />
+    <StrideNativeCPU Condition="$([MSBuild]::IsOSPlatform('Windows'))" Include="osx-x64"  LibraryExtension=".dylib" LibraryRuntime="osx" />
```

Comment:
Did you mean to use windows here ? and at line 22 ?

Reply:
@Eideren I had to restrict generating dlls and dylib only on Windows as a **temporary** workaround.
Two reassons: `CompileNativeClang_Windows` target tries to build using vcxproj - Visual C++ is supported only by msbuild NET FM and `CompileNativeClang_macOS` uses darwin-ld.exe (???????) - I didn't figure out where the heck I could find apple ld. Only clue I found is  [LLVM apple's fork ](https://github.com/apple/llvm-project).
To sum up, I don't want to break things yet but only allow to extend building for Linux for now.

Maybe I should add a comment here?

Reply:
Yes, please do

Reply:
Done! Hope that be useful for others :smile: